torch/csrc/jit/mobile/interpreter.cpp

```diff
@@ -93,6 +93,11 @@ bool InterpreterState::run(Stack& stack) {
       case SET_ATTR: {
         auto v = pop(stack);
         auto userObj = pop(stack).toObject();
+        // Mobile only: since the number of slots is not known, resize the numAttributes
+        // before setSlot.
+        while (userObj->type()->numAttributes() <= inst.X) {
```

Comment:
The combination of `while`, and `v.type()` is not correct. I think it would be best to just set all member types to `AnyType` rather than give them a wrong type.

Reply:
There's exception when I use AnyType. Should I disable `checkNoAny()`?

Reply:
I have changed it to NonType, which may be close to "neglecting type in loaded obj" as @zdevito suggested. I feel it's easier than introducing another #ifdef inside setslot(), where some codes may be hidden from unit test.